draft-ietf-rats-architecture.md

```diff
@@ -697,6 +697,13 @@ for by hardware or by ROM code, especially if such hardware is
 physically resistant to hardware tampering.  The component that is
 implicitly trusted is often referred to as a Root of Trust.
 
+A conveyance protocol that is authenticated and integrity protected can be used
+to convey unprotected Evidence, assuming the following properties exists:
+
+   1. The key used to authenticate and integrity protect the conveyance channel is trusted by the Verifier to speak for the Attesting Environment that collected claims about the Target Environment.
```

Comment:
keys? of at least two remote peers? We are talking about a "Secure Channel" to convey unprotected Evidence right?

Also? In theory, unprotected Evidence are just Claims?

I am not sure if we should water down evidence, in general. Evidence should be tamper-evident. In consequent, messing with evidence conveyed via "secure channel" must be evident with a similar level of assurance.